### PR TITLE
Modify the IE regex to include IE11.

### PR DESCRIPTION
**Pull request by Ujjwalreddy-okta**
OKTA-94262

### CODE REVIEW

**Comment by alexma-okta** [comment]
:rocket:

File: src/util/BrowserFeatures.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ define(function () {
   };
 
   fn.isIE = function () {
-    return /msie/i.test(navigator.userAgent);
+    return /msie/i.test(navigator.userAgent) || /trident/i.test(navigator.userAgent);
```
Comment (haishengwu-okta) [inline src/util/BrowserFeatures.js]: super picky: 
does this work `/(msie|trident)/.test(...)`?

Comment (lboyette-okta) [inline src/util/BrowserFeatures.js]: Is this how we want to check for IE at all? Why not see if document.documentMode exists?

Comment (Ujjwalreddy-okta) [inline src/util/BrowserFeatures.js]: @haishengwu-okta done.
@lboyette-okta There are a couple of ways to do this, but checking for the useragent string seems to be the most commonly used, so I went with it.

**Pull request merged**